Cargo.lock

```diff
@@ -32,9 +32,9 @@ checksum = "4d25d88fd6b8041580a654f9d0c581a047baee2b3efee13275f2fc392fc75034"
 
 [[package]]
 name = "arrayvec"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
I think we didn't add this after all, right? Better to stick with what's in the lockfile already in that case.

Reply:
Hmn..I removed arrayvec from Cargo.toml but the version inCargo.lock didn't revert back. How can that be done?

Reply:
Personally I'd copy over the file from main and then just do a quick `cargo run`+`cargo test` to make sure all is good.